react/features/whiteboard/middleware.any.ts

```diff
@@ -45,7 +45,6 @@ MiddlewareRegistry.register((store: IStore) => next => action => {
             store.dispatch(setupWhiteboard({
                 collabDetails: metadata[WHITEBOARD_ID].collabDetails
             }));
-            store.dispatch(setWhiteboardOpen(true));
```

Comment:
https://github.com/jitsi/jitsi-meet/assets/26413496/2e093088-ebd0-4ab0-b821-2bccfaabad8f

Why this approach? Why not leave it to the user to open/close the whiteboard whenever they want.

Reply:
Just leaving the reason here for posterity: other participants wouldn't otherwise get notified that a moderator has enabled the whiteboard. Maybe a notification with an action to take you to the whiteboard would fix the existing issue.

Reply:
No need. This change was discussed and reverted. All good.